### PR TITLE
Extra params for tasking manager

### LINKED ISSUE #343: Advanced Panel Design Edits
Let's have the Advanced panel be hidden by default:

**> Advanced** 

with an option to expand, i.e.,

**V Advanced**
[ Proxy URL options ]
[ Parameter options ]

### PR DESCRIPTION
**Pull request by kamicut**
Closes #343

### CODE REVIEW

**Comment by kamicut** [comment]
image

@LanesGood this is how the new component looks, kicking it to you for styling.

File: components/QueryParameters.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    }
+  }
+
+  onChange (params) {
+    this.props.onChange(params)
+  }
+
+  render () {
+    let params = Object.assign({}, this.props.params)
+    console.log(params)
```
Comment (alyssadelaine) [inline components/QueryParameters.js]:
```suggestion
```

**Comment by alyssadelaine** [comment]
I also get an error when I try to edit the parameters:
paramEx2

**Comment by alyssadelaine** [comment]
As far as the functionality goes, I tested:

✅ Single param works as expected
✅ Two params work as expected
✅ Compound param (i.e. mappingTypes =`ROADS,BUILDINGS`) works as expected

So I think once the above issues are addressed, this will be good to go

**Comment by kamicut** [comment]
@LanesGood this branch could use your 👁 
1. Review @alyssadelaine's suggestion from above
2. I think this component could use some styling

**Pull request merged**